tests/dsp/dsp_in_registered/README.rst

```diff
@@ -0,0 +1,37 @@
+DSP-style block with all inputs registered
+++++++++++++++++++++++++++++++++++++++++++
+
+This uses the model from |fig60|_ and the |dsp_combinational|_ module.
```

Comment:
The `|fig60|_` and `|dsp_combinational|_` doesn't seem to be rendering correctly?

Reply:
This is fixed.